include/envoy/upstream/host_description.h

```diff
@@ -84,9 +84,9 @@ class HostDescription {
   virtual const HostStats& stats() const PURE;
 
   /**
-   * @return the "zone" of the host (deployment specific). Empty is unknown.
+   * @return the locality of the host (deployment specific). This will be empty if unknown.
```

Comment:
nit: not really empty? Default object? (Not sure what default empty locality really means in object form)